modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx

```diff
@@ -93,6 +93,18 @@ const TestFlowTasks = () => {
 		return <Loading />;
 	}
 
+	const getAlert = () => {
+		if (selectedRows.length === 1) {
+			return [
+				{
+					text: i18n.translate(
+						'please-select-at-least-two-subtasks-to-merge'
+					),
+				},
+			];
+		}
```

Comment:
As you said that more alerts may appear, add it conditionally this way: 🦅 

```suggestion
               const alerts = [];
               
		if (selectedRows.length === 1) {
			alerts.push({
				text: i18n.translate(
					'please-select-at-least-two-subtasks-to-merge'
				)}
		}
		
		return alerts;
```